sashimi-webapp/src/logic/inputHandler/DocumentPrinter/index.js

```diff
@@ -0,0 +1,61 @@
+import elementUtils from 'src/helpers/elementUtils';
+import core from './core';
+
+const keyEventListener = function keyEventListener(event) {
+  if (event.key === 'p' && (event.ctrlKey || event.metaKey)) {
+    // metaKey will match Window key and Mac's CMD key
```

Comment:
Do you foresee any other issues with this part besides the window key on Windows? E.g. different browser compatibility? Cause I have been seeing this about the event.metaKey. (See first and second answers part)

http://stackoverflow.com/questions/3902635/how-does-one-capture-a-macs-command-key-via-javascript